db_service/db_handler.go

```diff
@@ -0,0 +1,15 @@
+package db_service
```

Comment:
Let's rename the pkg to `datastore` and insert the cache mechanism to here as well

Reply:
No, I think that the cache should be under the `datastore` pkg